package/src/components/Message/MessageSimple/ReactionList.tsx

```diff
@@ -250,25 +283,16 @@ const ReactionListWithContext = <
               styles.reactionBubble,
               {
                 backgroundColor: alignmentLeft ? fill : white,
-                borderRadius: reactionSize - strokeSize * 2,
+                borderRadius: reactionsData.extraSpace ? 5 : reactionSize,
```

Comment:
What is the need to adapt the border radius? it feels like it should be same regardless of having a counter or not

Reply:
In previous iterations icons would get misplaced out of the container, this no longer happens so I'll revert the changes on that line. Thanks

Reply:
<img width="288" alt="Captura de Tela 2023-09-06 às 21 57 12" src="https://github.com/GetStream/stream-chat-react-native/assets/3487334/cf7e95bb-1e84-4a87-b6c1-5b5fc3e2bc58">